health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AppPreview.js

```
@@ -325,10 +329,33 @@ const getFieldType = (field) => {
-  return (
+  const MODULE_CONSTANTS = "HCM-ADMIN-CONSOLE";
+  const componentMasterName = "RegistrationComponentsConfig";
+
+  const { isLoading: isLoadingComponentMaster, data: ComponentConfigMdmsData } = Digit.Hooks.useCustomMDMS(
+    Digit.ULBService.getCurrentTenantId(),
+    MODULE_CONSTANTS,
+    [
+      { name: componentMasterName, limit: 100 },
+    ],
+    {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)